tool/tctl/common/collection.go

```diff
@@ -83,6 +91,13 @@ type namespaceCollection struct {
 	namespaces []services.Namespace
 }
 
+func (c *namespaceCollection) resources() (r []services.Resource) {
+	for _, resource := range c.namespaces {
+		r = append(r, &resource)
```

Comment:
Might be missing something but does this code deliberately get the address with `&resource` while all other instances of `append(r, resource)` in this file aren't?

Reply:
Annoyingly, yes.  This is on purpose though; all the other resource types are interface types wrapping struct pointers, so they coerce to `services.Resource` directly.  The `services.Namespace` type is a concrete struct whose pointer coerces to `services.Resource`, so it has to be handled differently.